docs/en/token_endpoint.rst

```diff
@@ -226,7 +226,7 @@ UserInfo endpoint to get user attributes.
 ID Token
 ++++++++
 
-The ID Token is a JSON Web Token (JWT) that contains information on the user that has executed the authentication. The RPs MUST validate the ID Token.
+The ID Token is a signed JSON Web Token (JWT) that contains information on the user that has executed the authentication. The RPs MUST validate the ID Token.
```

Comment:
on the user that has executed the authentication --> about the user that has performed the authentication